### PR TITLE
fix pytest compatilbility

### PR DESCRIPTION
**Pull request by jardaburian**

### CODE REVIEW

File: src/python/test/test_dxclient.py [hunk 1]
```diff
@@ -128,12 +128,12 @@ def test_without_project_context(self):
...
-        self.assertIn('DX_SECURITY_CONTEXT', run('dx env --bash'))
-        with without_auth():
-            self.assertNotIn('DX_SECURITY_CONTEXT', run('dx env --bash'))
-        self.assertIn('DX_SECURITY_CONTEXT', run('dx env --bash'))
+    # @unittest.skipUnless(testutil.TEST_ENV, 'skipping test that would clobber your local environment')
+    # def test_without_auth(self):
+    #     self.assertIn('DX_SECURITY_CONTEXT', run('dx env --bash'))
+    #     with without_auth():
+    #         self.assertNotIn('DX_SECURITY_CONTEXT', run('dx env --bash'))
+    #     self.assertIn('DX_SECURITY_CONTEXT', run('dx env --bash'))
```
Comment (avigriyanova) [inline src/python/test/test_dxclient.py:136]: why commented not deleted?

Comment (jardaburian) [reply src/python/test/test_dxclient.py:136]: I asked - is this test important? nobody knows..

Comment (jardaburian) [reply src/python/test/test_dxclient.py:136]: you think I should better delet?

Comment (avigriyanova) [reply src/python/test/test_dxclient.py:136]: I think it's better to add a comment about it

Comment (jardaburian) [reply src/python/test/test_dxclient.py:136]: add a comment to a comment? NOISCE

**Pull request merged**